frontend/components/site/SiteSettings/AdvancedSiteSettings.jsx

```diff
@@ -117,16 +118,31 @@ export const AdvancedSiteSettings = ({
       Save advanced settings
     </button>
 
-    <AlertBanner
-      status="warning"
-      header="Danger zone"
-      message="Delete this site and its previews from Federalist, for all users?"
-      alertRole={false}
-    >
-      <button className="usa-button usa-button-red" onClick={onDelete}>
-        Delete
-      </button>
-    </AlertBanner>
+    <div className="well">
+      {/* DELETE SITE */}
+      <fieldset>
+        <legend>Delete Site</legend>
+        <p className="well-text">
+          Deleting a Federalist site removes the published site from our servers and
+          disconnects the Federalist admin interface for all users. This will bring the
+          entire site offline and make it inaccessible for users. <i>Trying to remove a site
+          from your list of Federalist sites? Go to the
+          {' '}
+            <a href={`/sites/${siteId}/users`}>Collaborators page</a> and remove yourself.</i>
```

Comment:
Do you mean for the `C` to be capitalized even though it is in the middle of a sentence?

Reply:
I'm asking in Slack: https://gsa-tts.slack.com/archives/C06RFLK9U/p1530046689000550